showcase/src/index.html

```diff
@@ -23,11 +23,6 @@
 		<!--Configured Head Tags  -->
 		<%= webpackConfig.htmlElements.headTags %> <% } %>
 
-		<!-- Stylesheets -->
-		<!-- The stylesheet should be loaded before angular is initialized -->
-		<link rel="stylesheet" href="/assets/stark-core/css/pre-load-style.css" />
-		<!-- Generated CSS stylesheets will be added by webpack: extract-text-webpack-plugin -->
```

Comment:
I think the comment in this line is still valid, just the plugin is different: it should be `mini-css-extract-plugin`